usr/lib/cca_stdll/cca_specific.c

```diff
@@ -1742,6 +1719,18 @@ static CK_RV cca_get_adapter_domain_selection_infos(STDLL_TokData_t *tokdata)
     }
     TRACE_DEVEL("num_adapters: %u\n", cca_private->num_adapters);
 
+#if defined(_AIX)
+   /*
+    * Short-circuit! If we're on Power, it means the card does not support CCA
```

Comment:
If we're on Power -> AIX

Well, on Linux on Power its certainly the same, but its within an ifdef AIX block, so I guess it should talk about AIX, not Power in general.